recipes/eigency/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "eigency" %}
+{% set version = "2.0.0a1" %}
+{% set eigenversion = "3.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/wouterboomsma/eigency/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 4c9a16adf1710ac647670a8a7516cca8e3f33201fedec9056268df1514b9552f
+
+build:
+  script: EIGEN_INC=${PREFIX}/include/eigen3 {{ PYTHON }} -m pip install --no-deps . -vv  # [not win]
+  script: set EIGEN_INC=%LIBRARY_INC%\eigen3 & {{ PYTHON }} -m pip install --no-deps . -vv  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cython
+    - numpy {{ numpy }}
+    - eigen {{ eigenversion }}
+  run:
+    - python
+    - numpy {{ numpy }}
```

```suggestion
    - numpy
    - eigen {{ eigenversion }}
  run:
    - python
    - {{ pin_compatible('numpy') }}
```
https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

---

Done I changed the lines for the pinning, Thank for all the comments. (it is the first time building a conda-forge recipe)